project-base/storefront/gtm/factories/getGtmTransportAndPaymentPageViewEvent.ts

```diff
@@ -13,5 +15,6 @@ export const getGtmTransportAndPaymentPageViewEvent = (
         valueWithVat: gtmCartInfo.valueWithVat,
         products: gtmCartInfo.products,
     },
+    user: mapGtmUserEntryInfoFromGtmUserInfo(user),
```

Comment:
I do not see the AC specifying this. Was it agreed upon?

Reply:
Mistaken for payment.